cpp/cmake_modules/SetupCxxFlags.cmake

```
@@ -28,7 +28,7 @@ if(NOT DEFINED ARROW_CPU_FLAG)
     set(ARROW_CPU_FLAG "armv8")
   elseif(CMAKE_SYSTEM_PROCESSOR MATCHES "armv7")
     set(ARROW_CPU_FLAG "armv7")
-  elseif(CMAKE_SYSTEM_PROCESSOR MATCHES "ppc")
+  elseif(CMAKE_SYSTEM_PROCESSOR MATCHES "powerpc|powerpc64|ppc")
```

It seems that `poerpc64` is redundant.

Does the following pattern work?

```suggestion
  elseif(CMAKE_SYSTEM_PROCESSOR MATCHES "powerpc|ppc")
```

---

Yes, that will work.

---

OK. I'll apply it.